post-processor/vagrant/libvirt.go

```diff
@@ -64,26 +63,23 @@ func (p *LibVirtProvider) KeepInputArtifact() bool {
 	return false
 }
 func (p *LibVirtProvider) Process(ui packersdk.Ui, artifact packersdk.Artifact, dir string) (vagrantfile string, metadata map[string]interface{}, err error) {
-	diskName := artifact.State("diskName").(string)
+	disks := []map[string]string{}
+	format := artifact.State("diskType").(string)
 
 	// Copy the disk image into the temporary directory (as box.img)
-	for _, path := range artifact.Files() {
-		if strings.HasSuffix(path, "/"+diskName) {
-			ui.Message(fmt.Sprintf("Copying from artifact: %s", path))
-			dstPath := filepath.Join(dir, "box.img")
-			if err = CopyContents(dstPath, path); err != nil {
-				return
-			}
+	for i, path := range artifact.Files() {
```

Comment:
Given the qemu builder only builds one disk, this should probably only include it, and the size provided by the artifact can be fetched as it was before, from the `diskSize` attribute.

Since there may be several files however (typically the `efivars.fd` for UEFI builds), I would keep the former check on the file itself so we only keep it in the provider's config.

Reply:
After rebase, i keep the check of the start of name of files.
If the file name start by 'VmName', it is a disk.

Reply:
Yes, it should not include the efivars that way, so that looks good to me, thanks